dbms/src/Common/StackTrace.cpp

```diff
@@ -62,6 +62,9 @@ std::string StackTrace::toString() const
         throw;
     }
 
+/// Dirty workaround for Asan false positive
```

Comment:
1. How exactly does it complain?
2. This is not the only place, look at three lines above.
3. Why the define is named BUILD_TYPE_Asan and not BUILD_TYPE_ASAN?
4. We don't need this define because we already have ADDRESS_SANITIZER define.

Reply:
1. Freeing of unallocated memory.
For some reason, it was reproduced on Ubuntu 17.10, but currently, it is not reproduced on 18.04.

Reply:
So, we can forgive about it.